packages/headless/esbuild.mjs

```diff
@@ -200,7 +200,6 @@ async function buildBrowserConfig(options, outDir) {
       alias({
         'coveo.analytics': resolveEsm('coveo.analytics'),
         pino: resolveBrowser('pino'),
-        '@coveo/pendragon': resolve('./ponyfills', 'magic-cookie-browser.js'),
```

Comment:
@louis-bompart I presume we no longer need the magic cookie

Reply:
As discussed in person, we'll keep it after all so I'll revert the cookie-related changes in this PR.